### PR TITLE
chore(release): pull release/v1.72.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3584/test-report.html

**Review by ItsSudip — APPROVED** [review]

File: src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "destKey": "channels.email.address",
+    "sourceKeys": "emailOnly",
```
Comment (krishna2020) [inline src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json:4]: @Gauravudia  how will `emailOnly` gets instrumented outside `traits` ?

**Review by krishna2020 — APPROVED** [review]

File: src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json (hunk 1 above)
Comment (Gauravudia) [reply src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json:4]: It is part of  "sourceFromGenericMap": true

Comment (Gauravudia) [reply src/cdk/v2/destinations/cordial/data/CordialIdentifyConfig.json:4]: https://github.com/rudderlabs/rudder-transformer/blob/20aa7f35e13ad89e8a43fbbb743df73b0c103975/src/v0/util/data/GenericFieldMapping.json#L6

**Pull request merged**